packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -0,0 +1,109 @@
+import { autorun } from 'mobx'
+import { addDisposer, getRoot, types } from 'mobx-state-tree'
+import _ from 'lodash'
+
+import * as helpers from '../features/feedback/shared/helpers'
+import strategies from '../features/feedback/shared/strategies'
```

Comment:
`helpers` and `strategies` should probably live somewhere under the store directory, rather than in a top-level `features` directory.

Reply:
That makes more sense to me too, I've updated #531 accordingly.